### DISCUSSION TITLE
Migrating CodedUI to APPIUM

### DISCUSSION THREAD

**Discussion by DivyaPonnala** [appium/appium#17574] [Q&A]
As CodedUI is deprecated and we have a large number of scripts written, is there a tool to convert CodedUI (Visual Studio, C#) to WinAppDriver or Appium? To rewrite these scripts manually would be a major development effort. Any suggestions are welcomed.

**Comment by jlipps** [comment]
No, there is no such tool that I am aware of. I would also not recommend migrating to WinAppDriver as Microsoft apparently abandoned it some years go.
